templates/package.xml.template

```diff
@@ -52,7 +52,7 @@
    <dependencies>
     <required>
      <php>
-      <min>5.5.0</min>
+      <min>7.0.0</min>
```

Comment:
Same here. Running `generate_projects.sh` will cause the real file `package.xml` to be updated. Both changes need to be checked in.